### ISSUE TITLE
pg_jobmon and dblink in cluster

### ISSUE THREAD

**Issue by sorquan** [omniti-labs/pg_jobmon#20]
Is it normal to install jobmon and point dblink to localhost on each node, or should I point dblink to the master?

**Comment by keithf4** [comment]
Yes, by default pg_jobmon is designed for it to connect back to itself. So it's not really made for jobs to be run on replicas and store their results.
